src/jit/simdintrinsiclist.h

```diff
@@ -134,6 +144,8 @@ SIMD_INTRINSIC("ShiftRightInternal",        false,       ShiftRightInternal,
 SIMD_INTRINSIC("UpperSave",                 false,       UpperSave,                "UpperSave Internal",     TYP_STRUCT,     2,      {TYP_UNDEF, TYP_UNDEF, TYP_UNDEF},     {TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF})
 SIMD_INTRINSIC("UpperRestore",              false,       UpperRestore,             "UpperRestore Internal",  TYP_STRUCT,     2,      {TYP_UNDEF, TYP_UNDEF, TYP_UNDEF},     {TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF})
 
+SIMD_INTRINSIC("WidenHi", false, WidenHi, "WidenHi", TYP_VOID, 3, { TYP_UNDEF, TYP_UNDEF,  TYP_UNDEF }, { TYP_INT, TYP_FLOAT, TYP_CHAR, TYP_UBYTE, TYP_BYTE, TYP_SHORT, TYP_UINT, TYP_UNDEF, TYP_UNDEF, TYP_UNDEF })
```

Comment:
Please also add comments explaining its semantics.

---
In reply to: [104520551](https://github.com/dotnet/coreclr/pull/9920#discussion_r104520551) [](ancestors = 104520551)